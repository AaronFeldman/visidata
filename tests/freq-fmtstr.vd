sheet	col	row	keystrokes	input	comment
			o	sample_data/sample.tsv	open file
sample	OrderDate		@		set type of current column to date
sample			C		open Columns Sheet
sample_columns	fmtstr	ã‚­sample,OrderDate	e	%Y	edit contents of current cell
sample_columns			q		quit current sheet
sample	OrderDate		F		open Frequency Table grouped on current column
sample_OrderDate_freq	histogram		-		hide current column
