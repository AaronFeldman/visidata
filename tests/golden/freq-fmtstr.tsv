Date	count	percent
2018-08	28	57.14
2018-07	21	42.86
